tests/main/connect-undo/test-connect.v2/bin/consumer

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+nc -h > /dev/null
```

Comment:
Nitpick, always use exec to have the application process without shell waiting to forward the exit code.

```suggestion
exec nc -h > /dev/null
```